### PR TITLE
Render mobile message replies

### PR DESCRIPTION
**Pull request by mxstbr**
**Status**
- [ ] WIP
- [x] Ready for review
- [ ] Needs testing

**TODO**

- [x] Render mobile message replies
  - [x] Collapse long quotes
  - [x] Fix width of quoted message
- [x] Style chat input
- [x] Render currently quoted message in chat input
- [x] Send message with parent ID
- [x] Add action UI for replies to message bubbles

**Current status:**

### CODE REVIEW

File: mobile/components/Message/style.js [hunk 1]
```diff
@@ -1,14 +1,52 @@
...
+  padding: ${props => (props.noPadding ? '0' : '8px 12px')};
+  margin-left: 4px;
+  margin-right: 4px;
+  margin-bottom: 0px;
+  margin-top: 4px;
+  overflow: hidden;
+  ${props => (props.expanded ? '' : 'max-height: 100px;')};
+`;
+
+export const QuoteGradient = styled(LinearGradient).attrs({
```
Comment (ryota-murakami) [inline mobile/components/Message/style.js:44]: oh, i don't know Expo provided `LinearGradient` & there is such as cool way to handle CSS Gradients by styled-components on ReactNative😋

**Review by ryota-murakami — DISMISSED** [review]

File: mobile/.expo/settings.json [hunk 2]
```diff
@@ -3,5 +3,5 @@
   "lanType": "ip",
   "dev": true,
   "minify": false,
-  "urlRandomness": "66-qcw"
+  "urlRandomness": "2s-bjc"
```
Comment (brianlovin) [inline mobile/.expo/settings.json:6]: Should this have notes around why this needs to be this specific thing?

**Comment by brianlovin** [comment]
Overall this looks good to me @mxstbr - I think you'll have the fastest ability to resolve those conflicts, then we can get this merged :)

File: mobile/.expo/settings.json (hunk 2 above)
Comment (mxstbr) [reply mobile/.expo/settings.json:6]: It's automatically generated by Expo

**Comment by mxstbr** [comment]
Will resolve and merge tomorrow!

**Review by brianlovin — DISMISSED** [review]

**Pull request merged**